.travis.yml

```diff
@@ -50,20 +50,25 @@ matrix:
   - env: EMBER_TRY_SCENARIO=ember-canary
 
 before_install:
-- "export DISPLAY=:99.0"
-- "sh -e /etc/init.d/xvfb start"
 - npm config set spin false
 - npm install -g coveralls pr-bumper
+- $(npm root -g)/pr-bumper/.travis/maybe-check-scope.sh
```

Comment:
`pr-bumper` was updated recently to support `#none#` as a version bump option, which won't actually publish a new version (useful for README and test updates). However in order to achieve this the way it works was changed a bit, in which it now makes an initial merge build that performs the version bump by making an automated commit on the repo. This updates the version in `package.json` as well as updates the `CHANGELOG.md`. Once that Travis build completes two additional Travis builds appear, one for the `master` branch and one for the new git tag, which `pr-bumper` creates for the new version. The build for the tagged branch is what actually publishes a new version to npm. This check is just an optimization that only checks for the presence of a bump commit in a PR for the initial merge build https://github.com/ciena-blueplanet/pr-bumper/blob/master/.travis/maybe-check-scope.sh.